lidarts/statistics/routes.py

```diff
@@ -28,35 +28,75 @@ def x01():
     user = User.query.filter(User.id == current_user.id).first_or_404()
 
     # if (form.is_submitted):
-    custom_filter_number_of_games = form.numberOfGames.default
-    custom_filter_date_from = form.dateFrom.default
-    custom_filter_date_to = form.dateTo.default
+    custom_filter_number_of_games = form.number_of_games.default
+    custom_filter_date_from = form.date_from.default
+    custom_filter_date_to = form.date_to.default
 
     # if no value is in the form data, we can insert the default values
-    if form.numberOfGames.data is None:
-        form.numberOfGames.data = custom_filter_number_of_games
-    if form.dateFrom.data is None:
-        form.dateFrom.data = custom_filter_date_from
-    if form.dateTo.data is None:
-        form.dateTo.data = custom_filter_date_to
+    if form.game_types.data is None:
```

Comment:
Brauchen wir die Checks überhaupt? Wenn ja:
form.game_types.data = form.game_types.default if not form.game_types.data else form.game_types.data 
sollte auch klappen

Reply:
Dachte aber bisher schon da beim initialen Laden die Daten nicht vorhanden sind. Oder bekomme diese dann direkt den Default Wert? Aber ich debugge das nochmal.